content/docs/db.json

```diff
@@ -413,6 +413,12 @@
     "contentPath": "./digging_deeper/i18n.md",
     "category": "Digging deeper"
   },
+  {
+    "permalink": "digging-deeper/cache",
+    "title": "Cache",
```

Comment:
The `Digging Deeper` section is alphabetically sorted. So we should move up this sub-section.